pkg/manager/member/ticdc_upgrader_test.go

```diff
@@ -82,8 +82,9 @@ func TestTiCDCUpgrader_Upgrade(t *testing.T) {
 			name: "normal",
 			expectFn: func(g *GomegaWithT, tc *v1alpha1.TidbCluster, newSet *apps.StatefulSet) {
 				g.Expect(tc.Status.TiCDC.Phase).To(Equal(v1alpha1.UpgradePhase))
-				g.Expect(newSet.Spec.UpdateStrategy.RollingUpdate.Partition).To(Equal(pointer.Int32Ptr(0)))
+				g.Expect(newSet.Spec.UpdateStrategy.RollingUpdate.Partition).To(Equal(pointer.Int32Ptr(1)))
```

Comment:
Why update this from `0` to `1`?
Because the Pod generated in L53 is not ready by default?
If this is the case, we'd better update that function to add condition true by default and add a case where the Pod is not ready.

Reply:
Please also add a similar case for the other components.